applications/luci-app-zerotier/root/etc/hotplug.d/iface/40-zerotier

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+zero_enable="$(uci get zerotier.sample_config.enabled)"
+nat_enable="$(uci get zerotier.sample_config.nat)"
+
+[ "$ACTION" = ifup -o "$ACTION" = ifupdate ] || exit 0
+[ "$ACTION" = ifupdate -a -z "$IFUPDATE_ADDRESSES" -a -z "$IFUPDATE_DATA" ] && exit 0
```

Comment:
This is hotplug stuff.
https://openwrt.org/docs/guide-user/base-system/hotplug
It sets variables and calls the script when events happen.

I'm removing it because I think it should be in the zerotier package if any